packages/third-parties/src/ThirdPartyScriptEmbed.tsx

```diff
@@ -15,6 +17,19 @@ export default function ThirdPartyScriptEmbed({
   children,
   dataNtpc = '',
 }: ScriptEmbed) {
+  useEffect(() => {
+    if (dataNtpc) {
+      // performance.mark is being used as a feature use signal. While it is traditionally used for performance
+      // benchmarking it is low overhead and thus considered safe to use in production and it is a widely available
+      // existing API.
+      performance.mark('mark_use_counter', {
```

Comment:
Working group changed their mind on naming:

```suggestion
      performance.mark('mark_feature_usage', {
```